embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/EmbraceDeliveryCacheManager.kt

```diff
@@ -228,21 +248,19 @@ internal class EmbraceDeliveryCacheManager(
     }
 
     private fun loadSession(cachedSession: CachedSession): SessionMessage? {
```

Comment:
This and the `override fun loadSession(String)` actually seem unused outside of test code